src/app/_components/OrganisationMembersGraphic.tsx

```diff
@@ -0,0 +1,277 @@
+      Math.PI / Math.asin(memberRadius / (7 * layerRadius))
+    )
+
+    //divide pi by people, if divided num too small -> add layers
+    if (people > host_layer1 + host_layer2) {
+      pi_division = (2 * Math.PI) / host_layer1
+      pi_division2 = (2 * Math.PI) / host_layer2
+      pi_division3 = (2 * Math.PI) / (people - host_layer1 - host_layer2)
+    } else if (people > host_layer1) {
+      pi_division = (2 * Math.PI) / host_layer1
+      pi_division2 = (2 * Math.PI) / (people - host_layer1)
+    } else {
+      pi_division = (2 * Math.PI) / people
+    }
+
+    //setting up layers
+    for (let i = 0; i < Math.min(people, host_layer1); i++) {
+      host_circles.push({
+        name: i,
+        cx: 85 + 6 * layerRadius * Math.sin(i * pi_division - theta),
+        cy: 85 + 6 * layerRadius * Math.cos(i * pi_division - theta),
+        r: memberRadius
+      })
+    }
+    for (let i = host_layer1; i < host_layer1 + host_layer2; i++) {
+      host_circles.push({
+        name: i,
+        cx: 85 + 7 * layerRadius * Math.sin(i * pi_division2 + theta),
+        cy: 85 + 7 * layerRadius * Math.cos(i * pi_division2 + theta),
+        r: memberRadius
+      })
+    }
+    for (let i = host_layer1 + host_layer2; i < people; i++) {
+      host_circles.push({
+        name: i,
+        cx: 85 + 8 * layerRadius * Math.sin(i * pi_division3 - theta),
+        cy: 85 + 8 * layerRadius * Math.cos(i * pi_division3 - theta),
+        r: memberRadius
+      })
+    }
+    //animating the circles
+    useEffect(() => {
+      const updateTheta = () => {
+        setTheta(prevTheta => (prevTheta + 0.001) % (2 * Math.PI))
+      }
+
+      const interval = setInterval(updateTheta, updateSpeed)
+
+      return () => clearInterval(interval)
+    }, [])
+
+    return host_circles
+  }
+
+  const pgCircles = PgCircleList(20).map(circle => {
+    return (
+      <circle
+        key={circle.name}
+        fill="red"
+        cx={circle.cx}
+        cy={circle.cy}
+        r={circle.r}
+      />
+    )
+  })
+
+  const otCircles = OtCircleList(80).map(circle => {
+    return (
+      <circle
+        key={circle.name}
+        fill="rgb(0 215 144)"
+        cx={circle.cx}
+        cy={circle.cy}
+        r={circle.r}
+      />
+    )
+  })
+
+  const hostCircles = HostCircleList(120).map(circle => {
+    return (
+      <circle
+        key={circle.name}
+        fill="rgb(0 100 70)"
+        cx={circle.cx}
+        cy={circle.cy}
+        r={circle.r}
+      />
+    )
+  })
+
+  return (
+    <figure className="relative mt-5 w-9/12 object-scale-down">
+      <svg viewBox="0 0 170 170" xmlns="http://www.w3.org/2000/svg">
+        <circle fill="orange" cx="85" cy="85" r={memberRadius} />
+        {pgCircles}
+        {otCircles}
+        {hostCircles}
+      </svg>
+      <figcaption className="bottom-0 opacity-90 md:absolute">
+        <OrganisationMembersInfo title="Infographics guide">
+          The graphics below contain our current organisation structure.
```

Comment:
We could change this to be more of a sales pitch! The idea of this graphic is to show how big and impressive armada is, so maybe something like "Each year more than 200 students join Armada to make the fair happen, each of these circles represents one of our members"